orion-ui/src/pages/TaskRun.vue

```diff
@@ -13,6 +13,17 @@
         <TaskRunLogs :task-run="taskRun" />
       </template>
 
+      <template v-if="taskRun" #task-inputs-heading>
+        Task inputs
+        <ExtraInfoModal title="Task Inputs">
+          <template #default>
+            <p>
+              Task inputs are passed to task runs and can
+              include constants, parameter keys, and other task runs.
```

Comment:
Hm except right now we are not including constants. It's always a mapping of _parameter keys_ to something the API has been told is to track as an input. Right now, the API is only tracking task run relationships that the client reports when it sees _results from other task runs_.

Reply:
OK.  I may need your input on this one.  Any good suggestions for the exact wording here?

Reply:
Updated and using localization for this to keep updates across oss and cloud simpler. Linked to https://github.com/PrefectHQ/orion-design/pull/1136